http-api/src/main/java/net/runelite/http/api/hiscore/HiscoreSkill.java

```diff
@@ -52,13 +52,14 @@
 	CONSTRUCTION("Construction"),
 	BOUNTY_HUNTER_HUNTER("Bounty Hunter - Hunter"),
 	BOUNTY_HUNTER_ROGUE("Bounty Hunter - Rogue"),
+	LAST_MAN_STANDING("Last Man Standing"),
```

Comment:
Why was this moved?

Reply:
Because LMS has changed positions in the Hiscore API and I wanted to let it reflect that.